README.md

```diff
@@ -1 +1,63 @@
-# new-years-countdown
\ No newline at end of file
+<h1 align="center"><strong> :santa: :christmas_tree: SECRET SANTA: A HOLIDAY HACKATHON :christmas_tree: :santa:</strong></h1>
+
+<h1 align="center"><strong> :champagne: New Year's Countdown :champagne: </strong></h1>
+
+## **Deployment**
+
+The website was deployed to GitHub pages:
+
+- From this project's [link for repository](https://github.com/nacht-falter/new-years-countdown), go to the **Settings**.
+- From the left-hand menu, click on the **Pages**.
+- Under the **Source** section, select the **Main** branch from the drop-down menu and click **Save**.
+- A message will appear to confirm a successful deployment to GitHub pages and provide the live link.
+
+The project is deployed and can be accessed here: [New Year's Countdown]()
+
+### Forking the GitHub Repository
+- Go to the GitHub repository.
+- In the top right hand corner you can click on the fork button.
+- This will fork the repository.
+
+## Intro
+Welcome to ***New Year's Countdown*** -  your go-to destination for ringing in the new year with style and excitement! As the clock ticks down to midnight, join us on a virtual journey filled with anticipation, joy, and the thrill of a fresh start. Our app is designed to make your New Year's Eve memorable, providing real-time countdowns, vibrant visuals, and a festive atmosphere that will get you in the celebratory spirit.
+
+Whether you're counting down with friends and family or embracing the moment solo, our app is your companion for the final seconds of the year. Get ready to bid farewell to the past and welcome the future with open arms.
+
+## **Planning Stage**
+
+### **Site Goals**
+
+- Objective(s)
+  - Create responsive website
+  - Create easy to navigate website
+- Problem Statement
+- Target Audience
+  - Anyone interested in celebrating New Year
+- Benefits
+
+### **Wireframes**
+
+For this project we made wireframes to help with planning and design. While some changes were made as we went along and ideas were added to our project, original wireframes can be found below:
+
+-
+
+## Tech
+The tech used in this project is as follow:
+* [Font Awesome](https://fontawesome.com/)
+* [Google Fonts](https://fonts.google.com/)
+* HTML 5 - Content and Structure
+* CSS 3 - Styling
+* Javascript - Interactive features
+* [Balsamiq](https://balsamiq.com/) - Wireframes
```

Comment:
I actually used Lucid Chart for the wireframes: https://www.lucidchart.com

Reply:
I'll change it with next push